tests/cases/eth2eth/scripts/init-rly

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -eu
+
+SCRIPT_DIR=$(cd $(dirname ${BASH_SOURCE:-$0}); pwd)
+source ${SCRIPT_DIR}/util/relayer-util
+
+rm -rf ${FIXTURES_DIR}
+rm -rf ${RELAYER_CONF} &> /dev/null
+
+sleep 10
+
+${RLY} config init
+${RLY} chains add-dir ${CONF_DIR}/relayer/demo/
+
+# add a path between chain0 and chain1
+$RLY paths add $CHAINID_ONE $CHAINID_TWO $PATH_NAME --file=${CONF_DIR}/relayer/path.json
+
+# handshake
+set -x
+$RLY tx clients $PATH_NAME
+$RLY tx update-clients $PATH_NAME
+$RLY tx connection $PATH_NAME
+$RLY tx channel $PATH_NAME
+set +x
+
```

Comment:
Can you split `init-rly` into `init-rly` and `handshake`?
c.f. https://github.com/datachainlab/yui-relayer-build/blob/v0.2/tests/cases/tm2tm/scripts

I think `fixture` is not necessary for this case.

Reply:
Split into init-rly and handshake
Remove fixture 
https://github.com/datachainlab/yui-relayer-build/pull/5/commits/9ad5aca503830fc1d084dc839397f39de10ad8bb